src/index.js

```diff
@@ -1,14 +0,0 @@
-import express, { json } from 'express';
-import cors from 'cors';
-import routes from './routes/routes.js'
-import dotenv from "dotenv";
-dotenv.config();
-
-const app = express();
-app.use(json());
-app.use(cors());
-app.use(routes);
-
-app.listen(process.env.PORT, () => {
-    console.log('Servidor online na porta ' + process.env.PORT);
-});
```

Comment:
Aqui é legal que a porta do servidor seja uma variável de ambiente. Além de ser uma boa prática, facilita para quem for rodar seu projeto posteriormente 

```
const port = process.env.PORT || 5000;
app.listen(port , () => ...
```